docs/api.md

```diff
@@ -66,9 +66,18 @@ Wrap the `HomeRoute` in an `AuthenticatedRoute` to specify the route you want to
 Route that when used, requires that a session is established before continuing. Else redirects the user to the `LoginRoute` path.
 
 ```html
-<AuthenticatedRoute path='/home/protected' component={RegisterPage} />
+<AuthenticatedRoute path='/profile' component={ProfilePage} />
 ```
 
+Specify the option `inGroup` to only allow users in a certain group to access the path.
+
+```html
+<AuthenticatedRoute path='/admin' inGroup="administrator" component={AdminPage} />
+<AuthenticatedRoute path='/admin/support' inGroup={["administrator", "support"]} component={SupportAdminPage} />
```

Comment:
I think that it should be clarified that the user must be in **both** of these groups and not just one of them.